src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -207,12 +207,21 @@ public bool TryGetTelemetryId(out Guid telemetryId)
 
                     // We convert the code fixes and refactorings to UnifiedSuggestedActionSets instead of
                     // SuggestedActionSets so that we can share logic between local Roslyn and LSP.
-                    var fixes = GetCodeFixes(
-                        supportsFeatureService, requestedActionCategories, workspace,
-                        document, range, addOperationScope, cancellationToken);
-                    var refactorings = GetRefactorings(
-                        supportsFeatureService, requestedActionCategories, workspace,
-                        document, selectionOpt, addOperationScope, cancellationToken);
+                    var (fixes, refactorings) = ThreadingContext.JoinableTaskFactory.Run(async () =>
+                    {
+                        await ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+
+#pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task (this method manages affinity using JTF)
+                        var fixes = await GetCodeFixesAsync(
+                            supportsFeatureService, requestedActionCategories, workspace,
+                            document, range, addOperationScope, cancellationToken);
```

Comment:
can you just use .ConfigureAwait(true).  that's self documenting, extremely clear, and avoids the suppression.